tasks/2017/issues.md

```diff
@@ -10,34 +10,46 @@ completing the
 [Intro to Zulip Server development beginner task](https://github.com/zulip/zulip-gci/blob/master/tasks/intro-to-zulip-server.md)
 before claiming a GitHub issue task, since most GitHub issues require
 the knowledge you learn with that task.
+* If you are interested in working on a issue in some other project like the
+mobile or desktop app, we recommend going through the contribution and setup
+guide in their specific repository.
```

Comment:
This is fine for v1, but we might want to make some basic "get started" tasks for the other repos.  I opened https://github.com/zulip/zulip-gci/issues/615 for that.